packages/dd-trace/src/config.js

```diff
@@ -404,7 +408,7 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       maxConcurrentRequests: DD_IAST_MAX_CONCURRENT_REQUESTS,
       maxContextOperations: DD_IAST_MAX_CONTEXT_OPERATIONS
     }
-
+    this.remoteConfigEnabled = isTrue(DD_REMOTE_CONFIG_ENABLED)
```

Comment:
In that PR merged today (https://github.com/DataDog/dd-trace-js/pull/2675/files), we have a new object in config, `this.remoteConfig` and there we have a configurable property, `pollInterval`. I think that `this.remoteConfigEnabled` should become `this.remoteConfig.enabled`

Reply:
Updated accordingly. Thank you.